### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by JoshuaDelosSantos**
Hi @Nathan-Dachille can you please review my variable naming and SRP! 👹 
Please don't mind the prac_04 commits 🙈
Codes for prac_05 starts at "Add state names program"

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        if choice != "Y" and choice != "":
+            new_name = input("Name: ")
+            email_to_name[email] = new_name
+        email = input("Email: ")
+
+    for email in email_to_name:
+        print(f"{email_to_name[email].title()} ({email})")
+
+
+def extract_name(email):
```
Comment (Nathan-Dachille) [inline prac_05/emails.py:27]: FYI: Adding a docstring for this function would help with readability

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    parts = email.split('@')
+    possible_name = parts[0]
+    if "." in possible_name:
+        name = possible_name.split(".")
+        return " ".join(name).title()
+    return parts[0].title()
+
+
+main()
```
Comment (Nathan-Dachille) [inline prac_05/emails.py:34]: Overall, great variable and function names!

**Review by Nathan-Dachille — COMMENTED** [review]
Great job with the practical :+1:,

A couple very minor errors, those being one missing docstring in the first exercise; As well as missing newlines at the ends of the programs. Other then that you have done an excellent job, especially with variable naming!

Kind regards, Nathan.

File: prac_05/emails.py (hunk 1 above)
Comment (JoshuaDelosSantos) [reply prac_05/emails.py:27]: Ty kind Sir, done!

**Comment by JoshuaDelosSantos** [comment]
Thank you @Nathan-Dachille for the feedback in #2  👹

**Pull request merged**